tribits/devtools_install/install-cmake.py

```diff
@@ -45,7 +45,7 @@
 #
 
 cmakeBaseName = "cmake"
-cmakeDefaultVersion = "3.13.4"
+cmakeDefaultVersion = "3.17.0"
```

Comment:
For now I think we should keep the default version at 3.13.4 and then add an if-statement based on version to apply the patch for CMake 3.17.0.